src/App.jsx

```diff
@@ -419,6 +463,26 @@ function App() {
               >
                 Sign In
               </AnimatedButton>
+              <Divider/>
+              <AnimatedButton
+                type="submit"
+                onClick={signInWithGoogle}
+                variant="contained"
+                color="primary"
+                sx={buttonStyle}
+              >
+                Sign In With Google
```

Comment:
instead of this long dialogue, I believe we can use google logo as it will help in alignment as well with other method on same line

Reply:
@narayan954 yes, I was thinking the same but how about raise another issue for this? It'll make a good first issue. Let me know what do you think about it. 😃

Reply:
it works for me!